docs/snippets_table.py

```diff
@@ -203,6 +203,11 @@ def test_bigtable_write_read_drop_truncate():
     # Read full table
     partial_rows = table.read_rows()
     read_rows = [row for row in partial_rows]
```

Comment:
No need to convert to a list, just iterate below, matching the [example in the API docs](https://cloud.google.com/bigtable/docs/samples-python-hello#scanning_all_table_rows).